vmware/setup.sh

```diff
@@ -1,4 +1,4 @@
-kubectl apply -f ./vmware/vcsim_deployment.yml
+kubectl apply -f ./vmware/vcsim_deployment_http.yml
```

Comment:
i would add this steps to make the proccess more automatic
```bash
wget https://raw.githubusercontent.com/vmware/govmomi/master/Dockerfile.vcsim
curl -L -o - https://github.com/vmware/govmomi/releases/latest/download/vcsim_$(uname -s)_$(uname -m).tar.gz | tar -xvzf - vcsim
docker build -f ./vmware/Dockerfile.vcsim -t localhost:5001/vcsim .
docker push localhost:5001/vcsim

```

Reply:
AFAIK the vcsim does not have any disk image in the container image. So we will need to build it with it.
So we can't use the Dockerfile from vcsim master.

Reply:
I guess vcsim is pretty stable but still I wouldn't take the latest version on each run.. wouldn't it be better to build the container one in a while and push it to quay?